app/config/sql/user/delete.sql

```diff
@@ -0,0 +1 @@
+DELETE FROM users WHERE id=$1;
```

Comment:
Remove this for now, since this sql function will be used to delete a sentence, not a user. Cool?